runtime/parachains/src/configuration.rs

```diff
@@ -1005,6 +1020,7 @@ pub mod pallet {
 			new: u32,
 		) -> DispatchResult {
 			ensure_root(origin)?;
+			ensure!(new <= T::HrmpMaxOutboundChannelsBound::get(), Error::<T>::InvalidNewValue);
```

Comment:
ensuring the conditions in setters are now discouraged in favor of checking it centrally in `fn check_consistency`

Reply:
I settled this in https://github.com/paritytech/polkadot/pull/3876/commits/e22a79fd6afe8c955a96d0646071aea22017fc70, LMK what you think.

Reply:
So, I will also change this in Ump? #3889

Reply:
i missed your comment... ༼ ༎ຶ ෴ ༎ຶ༽ 

But that seems to be the same cases